client/contracts/oracle/gen_manager.go

```diff
@@ -7,17 +7,17 @@ import (
 	"math/big"
 	"strings"
 
-	"github.com/kowala-tech/kcoin/client/accounts/abi"
-	"github.com/kowala-tech/kcoin/client/accounts/abi/bind"
-	"github.com/kowala-tech/kcoin/client/common"
-	"github.com/kowala-tech/kcoin/client/core/types"
+	"github.com/ethersphere/go-ethereum/accounts/abi"
```

Comment:
I think the generator have broken our imports.
Unless this bug all is fine for me